.github/workflows/container-image.yml

```diff
@@ -70,13 +70,27 @@ jobs:
     - uses: actions/checkout@v3
     - name: Build the client image
       run: make build-client
+    # Here we upload samba-client image to artifacts locally for consumption
+    # during samba-toolbox build process.
+    - name: Upload client image
+      uses: ishworkh/docker-image-artifact-upload@v1
```

Comment:
Looks ok, but keep in mind that this [github-action](https://github.com/ishworkh/docker-image-artifact-download) has rather low activity (makes me wonder if it is something we would want to relay on). That said, I do now know of any better alternative.